src/deltacode/models.py

```diff
@@ -1,314 +0,0 @@
-            ('sha1', self.sha1),
-            ('fingerprint', self.fingerprint),
-            ('original_path', self.original_path),
-        ])
-
-        if self.licenses:
-            d['licenses'] = [l.to_dict() for l in self.licenses]
-        else:
-            d['licenses'] = []
-
-        if self.copyrights:
-            d['copyrights'] = [l.to_dict() for l in self.copyrights]
-        else:
-            d['copyrights'] = []
-
-        return d
-
-    def size_difference(self, other_file):
-        """
-        Return the difference between the size of the instant File object and
-        a second File object to which it is being compared.
-        """
-        return self.size - other_file.size
-
-    def __repr__(self):
-        """
-        Return string containing a printable representation of the File object.
-        """
-        return "%s" % self.__dict__
-
-
-class License(object):
-    """
-    License object created from the 'license' field in an ABCD formatted 'file'
-    dictionary.
-    """
-    def __init__(self, dictionary={}):
-        self.key = dictionary.get('key')
-        self.score = dictionary.get('score')
-        self.short_name = dictionary.get('short_name')
-        self.category = dictionary.get('category')
-        self.owner = dictionary.get('owner')
-
-    def to_dict(self):
-        """
-        Given a License object, return an OrderedDict with the full
-        set of fields from the ScanCode 'license' value.
-        """
-        d = OrderedDict([
-            ('key', self.key),
-            ('score', self.score),
-            ('short_name', self.short_name),
-            ('category', self.category),
-            ('owner', self.owner)
-        ])
-
-        return d
-
-    def __repr__(self):
-        """
-        Return string containing a printable representation of the License
-        object.
-        """
-        return "%s" % self.__dict__
-
-
-class Copyright(object):
-    """
-    Copyright object created from the 'copyrights' field in an ABCD formatted
-    'file' dictionary.
-    """
-    def __init__(self, dictionary={}):
-        self.statements = dictionary.get('statements')
-        self.holders = dictionary.get('holders')
-
-    def to_dict(self):
-        """
-        Given a Copyright object, return an OrderedDict with the full
-        set of fields from the ScanCode 'copyrights' value.
-        """
-        d = OrderedDict([
-            ('statements', self.statements),
-            ('holders', self.holders)
-        ])
-
-        return d
-
-    def __repr__(self):
-        """
-        Return string containing a printable representation of the Copyright
-        object.
-        """
-        return "%s" % self.__dict__
-
-
-class ScanException(Exception):
-    """
-    Named exception for JSON file (1) containing no 'scancode_version'
-    attribute, (2) containing old version of ScanCode, or (3) containing no
-    'scancode_options'/'--info' attribute.
-    """
-    pass
```

Comment:
This may be needed?

Reply:
Actually they are inbuilt in `VirtualCodebase` so i felt they might not be needed